.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "ext/moderngpu"]
 	path = ext/moderngpu
-	url = https://ctcyang@github.com/ctcyang/moderngpu.git
+	url = git@github.com:ctcyang/moderngpu.git
```

Comment:
nit: Kind of a personal preference thing, but could you change these to use HTTPS verification rather than SSH? I usually don't have ssh-key installed, so I typically use HTTPS.

I've tried with HTTPS and if we have access to GALATIC private repo, then so long as we enter our Github login, it lets you pull in the submodule.